.github/workflows/code-hygiene.yml

```diff
@@ -1,6 +1,6 @@
 name: Code Hygiene
 
-on: [push, pull_request]
```

Comment:
I searched for different events and went through different activity types in [pull_request](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) event but didn't fine one which triggers workflow on commit; while [push](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push) specifically does that. Am I missing something ?

Reply:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
* synchronize